tests/e2e/test_e2e_template.py

```diff
@@ -314,6 +393,32 @@ def _test_make_run_something_useful(target: str, path: str, timeout_run: int) ->
     wait_job_change_status_to(job_id, "succeeded")
 
 
+@pytest.mark.run(order=STEP_KILL)
+@try_except_finally(f"neuro kill {MK_TRAINING_JOB}")
+def test_make_connect_train_kill_train() -> None:
+    cmd = f"make train TRAINING_COMMAND='sleep 3h'"
+    with measure_time(cmd):
+        run(
+            cmd,
+            verbose=True,
+            detect_new_jobs=True,
+            expect_patterns=[_get_pattern_status_running()],
+            allow_nonzero_exitcode=True,
+        )
+    cmd = "make kill-train"
+    with measure_time(cmd):
+        run(cmd, detect_new_jobs=False)
+
+
+@pytest.mark.run(order=STEP_KILL)
+@try_except_finally()
+def test_make_kill_all() -> None:
+    # just check exit code
+    cmd = f"make kill-all"
+    with measure_time(cmd):
+        run(cmd, verbose=True, detect_new_jobs=False)
```

Comment:
I think, there's no need to run a job here as all other `make kill` commands are tested separately. Here, we need to check just that `make kill` does not fail.